### DISCUSSION TITLE
Kalman filtering adaptive MCMC

### DISCUSSION THREAD

**Discussion by ben18785** [pints-team/pints#1564] [New methods]
Interesting [paper](https://arxiv.org/pdf/1308.5875.pdf) by Haario (a profligate author of popular adaptive MCMC routines) introducing a Kalman filter adaptive MCMC routine. Their method uses a Kalman filter to derive optimal updates of the covariance matrix as an MCMC sampler runs.

**Comment by MichaelClerx** [comment]
Any relation to @martinjrobins ideas for a kalman-filter loglikelihood #433  ?

**Comment by ben18785** [comment]
Nope, is different. May use similar machinery though.
